extensions/panache/mongodb-panache-kotlin/runtime/src/main/kotlin/io/quarkus/mongodb/panache/kotlin/PanacheMongoEntityBase.kt

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.mongodb.panache.kotlin
+
+import io.quarkus.mongodb.panache.kotlin.runtime.KotlinMongoOperations
+
+/**
+ * Represents an entity. If your Mongo entities extend this class they gain auto-generated accessors
+ * to all their public fields, as well as a lot of useful methods. Unless you have a custom ID strategy, you
+ * should not extend this class directly but extend [PanacheMongoEntity] instead.
+ *
+ * @see PanacheMongoEntity
+ */
+@Suppress("unused")
+abstract class PanacheMongoEntityBase {
+    companion object {
+        @JvmStatic
+        val operations = KotlinMongoOperations()
+    }
+
+    /**
+     * Persist this entity in the database.
+     * This will set it's ID field if not already set.
+     *
+     * @see .persist
+     * @see .persist
+     * @see .persist
```

Comment:
These are weird artifacts of the conversion to kotlin.  I'll need to rewrite them in dokka.  But if I can pull off that common interface definition between java and kotlin, this would all go away.